aurorawatch-pipeline/conftest.py

```diff
@@ -0,0 +1,50 @@
+'''Contains the configurations for the tests in this folder.'''
+
```

Comment:
The AuoraWatch API is relatively simple as so is  limited in what it can return, these tests cover all those bases effectively.